Wikipedia/UI-V5/UIButton+WMFButton.m

```diff
@@ -100,8 +100,8 @@ - (void)wmf_setButtonType:(WMFButtonType)type {
         case WMFButtonTypeBookmarkMini:
             [self setImage:[UIImage imageNamed:@"save-mini"] forState:UIControlStateNormal];
             [self setImage:[UIImage imageNamed:@"save-filled-mini"] forState:UIControlStateSelected];
-            [self setTitle:@"Save for later" forState:UIControlStateNormal];
-            [self setTitle:@"Saved for later" forState:UIControlStateSelected];
+            [self setTitle:MWLocalizedString(@"button-save-for-later", nil) forState:UIControlStateNormal];
+            [self setTitle:MWLocalizedString(@"button-saved-for-later", nil) forState:UIControlStateSelected];
```

Comment:
this is ok, but in the future, can we make these kinds of keys more visually distinct? 1-letter differences can be very hard to spot